docker-compose.yml

```diff
@@ -32,7 +32,7 @@ services:
       resources:
         limits:
           cpus: "0.1"
-          memory: 400M
+          memory: 1024M
```

Comment:
docker compose is not executed in Azure.
We need to increase the limit in `createApp.js` file in central-api
currently the limit is: `app.use(express.json({ limit: '500kb' }));`
Can we increase it to 20mb to see what happens? only for Dev env first

Reply:
Sorry, have pushed.